File: MPI/mpicheck/tests/functions/1.0/MPI_Group_union.f

        
        program main
        include 'mpif.h'
        
        INTEGER var_0
       INTEGER var_1
       INTEGER var_2
       INTEGER var_3
        call mpi_group_union(var_0, var_1, var_2, var_3)
       call pmpi_group_union(var_0, var_1, var_2, var_3)
        end program main
    

File: MPI/mpicheck/tests/functions/1.0/MPI_Group_union.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Group var_0;
    MPI_Group var_1;
    MPI_Group *var_2;
    int ret;
    /* calls */
    ret = MPI_Group_union(var_0, var_1, var_2);
    ret = PMPI_Group_union(var_0, var_1, var_2);
    return 0;
}
